protected/widgets/views/GenericAdmin.php

```diff
@@ -93,8 +93,8 @@
 			<?php }?>
 		</table>
 		<div>
-            <?php if (!$this->cannot_add) { echo EventAction::button('Add', 'admin-add', null, array('class' => 'generic-admin-add small secondary', 'data-model' => $model, 'data-new-row-url' => @$this->new_row_url))->toHtml(); }?>&nbsp;
-			<?php echo EventAction::button('Save', 'admin-save', null, array('class' => 'generic-admin-save small primary'))->toHtml()?>&nbsp;
+        <?php if (!$this->cannot_add) { echo EventAction::button('Add', 'admin-add', null, array('class' => 'generic-admin-add small secondary', 'data-model' => $model, 'data-new-row-url' => @$this->new_row_url))->toHtml(); }?>&nbsp;
```

Comment:
It doesn't look like anything but whitespace changed here. In that case, you might want to revert these changes, then 
git commit --amend --no-edit 
and force push to fix the commit on the remote